scripts/lint-datagrid.test.sh

```
@@ -0,0 +1,346 @@
+NC='\033[0m' # No Color
+
+# Counters
+TESTS_RUN=0
+TESTS_PASSED=0
+TESTS_FAILED=0
+
+# Cleanup function
+cleanup() {
+  cd "$ORIGINAL_DIR"
+  rm -rf "$TEST_DIR"
+}
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)